Library/Formula/hyperscan.rb

```diff
@@ -19,7 +19,7 @@ class Hyperscan < Formula
   def install
     mkdir "build" do
       system "cmake", "..", *std_cmake_args
-      system "make", "install"
+      system "make", "install", "-j", "8"
```

Comment:
Please don't hardcore the make jobs. Use `"-j#{ENV.make_jobs}"` if you must, but otherwise don't add it.